common-util/prop-types.js

```diff
@@ -1,20 +1,59 @@
 import PropTypes from 'prop-types';
 
+const Voter = PropTypes.shape({
+  address: PropTypes.string,
+  signature: PropTypes.string,
+  votingPower: PropTypes.string, // passed in wei
+});
+
+const ProposerPropTypes = PropTypes.oneOfType([
+  PropTypes.string, // TODO: remove once backend is migrated
```

Comment:
We'll likely need a follow up PR for some of the UX updates as well as fixes and post-audit fixes. So we remove this line at that point.